src/main/java/org/jenkinsci/plugins/github/config/GitHubPluginConfig.java

```diff
@@ -171,7 +171,7 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
 
     @Override
     public String getDisplayName() {
-        return "GitHub Plugin Configuration";
+        return "GitHub Configuration";
```

Comment:
> it [is] extremely critical to know what plugin contributes what configuration

But it is precisely the point of this PR to make this configuration be more readily _applicable_ to multiple plugins, regardless of which plugin _contributes_ it.

From the user perspective, they are configuring GitHub settings in Jenkins. If they need to troubleshoot why something is not working as expected, there are multiple techniques for doing that, such as enabling logging, searching known bug reports, etc. The link in help text may indeed be used to determine which plugin is defining a given configuration fragment, but this is not necessarily the source of a problem anyway.